File: frontend/src/models/WorksheetLog.ts
export interface WorksheetLog {
  id: number;
  worksheetId: string;
  parameterCode: string | null;
  parameterName: string | null;
  timestamp: string;
  remarks: string | null;
  action: string;
  employeeId: string;
  employeeName: string;
  role: string;
  referenceType: string | null;
  referenceId: string | null;
}

File: backend/RawDataWorkSheet/RawDataWorkSheet/Models/Worksheets/WorksheetLog.cs
namespace RawDataWorkSheet.Models.Worksheets
{
    public class WorksheetLog
    {
        public int Id { get; set; }
        public string? WorksheetId { get; set; }
        public int? ParameterId { get; set; }
        public string? ParameterCode { get; set; }
        public string? ParameterName { get; set; }
        public DateTime Timestamp { get; set; }
        public string? Remarks { get; set; }
        public string? Action { get; set; }
        public string? EmployeeId { get; set; }
        public string? EmployeeName { get; set; }
        public string? Role { get; set; }
        public string? ReferenceType { get; set; }
        public string? ReferenceId { get; set; }
    }
}
